invoice/src/main/java/org/killbill/billing/invoice/template/formatters/DefaultInvoiceFormatter.java

```diff
@@ -71,6 +72,18 @@
     private final InternalTenantContext context;
     private final ResourceBundleFactory bundleFactory;
 
+    public static Map<java.util.Currency, Locale> currencyLocaleMap;
```

Comment:
What I meant was that if `currencyLocaleMap ` becomes an instance variable of `DefaultInvoiceFormatterFactory`, you'll need to change the constructor of `DefaultInvoiceFormatter` to pass it as an argument (so it's available in `DefaultInvoiceFormatter`).

`DefaultInvoiceFormatterFactory` however should still have a no-argument constructor.